eslint.config.mjs

```diff
@@ -0,0 +1,27 @@
+export default [
+  {
+    // Define language options
+    languageOptions: {
+      sourceType: 'module', // or 'commonjs' based on your project
+      globals: {
+        // Add any global variables you need
+        __dirname: true,
+        process: true,
+      },
+    },
+    rules: {
+      'eol-last': 'error',
+      'eqeqeq': ['error', 'allow-null'],
+      'indent': ['error', 2, { MemberExpression: 'off', SwitchCase: 1 }],
+      'no-trailing-spaces': 'error',
+      'no-unused-vars': [
+        'error',
+        {
+          vars: 'all',
+          args: 'none',
+          ignoreRestSiblings: true,
+        }
+      ],
+    },
+  },
+];
```

Comment:
I think that the config will require a team agrement.

Reply:
After i upgrade eslint, it need additional file config, and no need eslintrc yml, eslintignore. eslint config is the result converted from eslintrc, and i dont add rules or changes rule.